src/claude-session-utility.ts

```
@@ -1838,11 +1858,12 @@ export const loadSessionEnrichment = async (
         // from an in-process tail read (~256KB reaches far beyond the old
         // 50-line window — an active session's tail is often tool output with
         // no gitBranch field: measured tail -5 hit 0, tail -20 hit 12).
-        const [titleOutput, aiTitleOutput, prLinkOutput, tailOutput] =
+        const [titleOutput, aiTitleOutput, prLinkOutput, recapOutput, tailOutput] =
```

ESLint

[error] Replace `titleOutput,·aiTitleOutput,·prLinkOutput,·recapOutput,·tailOutput]·=` with `⏎··········titleOutput,⏎··········aiTitleOutput,⏎··········prLinkOutput,⏎··········recapOutput,⏎··········tailOutput,`

(prettier/prettier)